src/util/string.cpp

```diff
@@ -4,11 +4,12 @@
 
 #include <util/string.h>
 
-#include <boost/algorithm/string/replace.hpp>
-
+#include <regex>
 #include <string>
+#include <utility>
 
-void ReplaceAll(std::string& in_out, std::string_view search, std::string_view substitute)
+void ReplaceAll(std::string& in_out, const std::string& search, const std::string& substitute)
 {
-    boost::replace_all(in_out, search, substitute);
+    if (search.empty()) return;
+    in_out = std::regex_replace(in_out, std::regex(std::move(search)), substitute);
```

Comment:
you can't move a `const` value. This is a no-op. Even if it wasn't const, this would still be a no-op as there is no constructor taking a non-const ref

Reply:
#25872